### PR TITLE
Only update changed data 

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/7533

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-user-management/src/views/Users.vue [hunk 1]
```diff
@@ -409,9 +411,27 @@ export default defineComponent({
     },
     async editUser(editUser) {
       try {
-        await this.graphClient.users.editUser(editUser.id, editUser)
+        const actualUser = this.users.find((user) => user.id === editUser.id)
 
-        if (editUser.drive) {
+        const graphEditUserRawObjectExtractor = (user) => {
```
Comment (JanAckermann) [inline packages/web-app-user-management/src/views/Users.vue:416]: ehhhhhhhhhhhhhhhhhh

File: packages/web-app-user-management/src/views/Users.vue [hunk 2]
```diff
@@ -427,25 +447,30 @@ export default defineComponent({
...
-          }
-        )
+          )
+        }
 
-        const user = this.users.findIndex((user) => user.id === editUser.id)
-        this.$set(this.users, user, editUser)
+        this.$set(
+          this.users,
+          this.users.findIndex((user) => user.id === editUser.id),
```
Comment (kulmann) [inline packages/web-app-user-management/src/views/Users.vue:471]:
```suggestion
          actualUser,
```
no?

Comment (JanAckermann) [reply packages/web-app-user-management/src/views/Users.vue:471]: nope, $set requires the index

File: packages/web-app-user-management/src/views/Users.vue [hunk 3]
```diff
@@ -409,9 +411,27 @@ export default defineComponent({
     },
     async editUser(editUser) {
       try {
-        await this.graphClient.users.editUser(editUser.id, editUser)
+        const actualUser = this.users.find((user) => user.id === editUser.id)
 
-        if (editUser.drive) {
+        const graphEditUserRawObjectExtractor = (user) => {
+          const extractUser = { ...user }
```
Comment (kulmann) [inline packages/web-app-user-management/src/views/Users.vue:417]: probably easier to use `lodash-es/omit` for that :-)

Comment (JanAckermann) [reply packages/web-app-user-management/src/views/Users.vue:417]: <3

File: packages/web-app-user-management/src/views/Users.vue (hunk 2 above)
Comment (kulmann) [reply packages/web-app-user-management/src/views/Users.vue:471]: Ah, sorry. Didn't read carefully enough

**Review by kulmann — APPROVED** [review]
💪

**Pull request merged**